builtin/exec/platform.go

```diff
@@ -152,7 +152,7 @@ func (p *Platform) renderTemplate(tpl *ConfigTemplate, data *tplData) (string, f
 	if err != nil {
 		return "", nil, err
 	}
-	closer := func() {} //os.RemoveAll(td) }
+	closer := func() { os.RemoveAll(td) }
```

Comment:
Note that we don't check the error returned here. I'm not totally sure how we would handle that error anyway though. At least it's in /tmp and will be cleaned up by the OS at some point, which is what we're relying on now anyway by not cleaning up